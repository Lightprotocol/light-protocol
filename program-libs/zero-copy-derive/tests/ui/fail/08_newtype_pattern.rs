// Edge case: Newtype pattern
#![cfg(feature="mut")] use light_zero_copy_derive::{ZeroCopy, ZeroCopyMut};

#[derive(ZeroCopy, ZeroCopyMut)]
#[repr(C)]
pub struct NewType(pub Vec<u8>);

fn main() {}