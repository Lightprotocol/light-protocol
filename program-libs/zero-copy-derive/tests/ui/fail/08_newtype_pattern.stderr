error: ZeroCopy only supports structs with named fields
 --> tests/ui/fail/08_newtype_pattern.rs:6:19
  |
6 | pub struct NewType(pub Vec<u8>);
  |                   ^^^^^^^^^^^^^
